src/fetch.js

```diff
@@ -25,7 +25,61 @@ let requests = {
       .then(response => response.json())
       .then(data => data.sleepData)
       .catch(error => console.log(error))
-  }
+  },
```

Comment:
Little comma hiding out here

Reply:
That one does have to be there, as it is a method on the object 'requests'. But there is another one after the last method that is optional!